src/api/goods.ts

```diff
@@ -10,10 +10,13 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort((
+      a: Good, b: Good,
+    ) => a.name.localeCompare(b.name)))
+    .then(goods => goods.slice(0, 5));
```

Comment:
Sort already know about that
```suggestion
    .then(goods => goods.sort((firstGood, secondGood) => (
      firstGood.name.localeCompare(secondGood.name)
    ))
    .then(goods => goods.slice(0, 5));
```